src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -66,6 +66,16 @@
   </span>
 </ng-template>
 
+<ng-template #statusLight
```

Comment:
I wonder if a single colored circle without further context might not be confusing to a user. How is he/she supposed to know what this means? It would probably make sense to have at least some form of mouseover tooltip that explains what a particular color code means?

Reply:
Thanks, @LenzGr . I will try to add tooltip that explains what a particular color code means.